mlrun/platforms/other.py

```diff
@@ -165,3 +167,87 @@ def _mount_hostpath(task):
         )
 
     return _mount_hostpath
+
+
+def mount_s3(
+    secret_name=None,
+    aws_access_key="",
+    aws_secret_key="",
+    endpoint_url=None,
+    prefix="",
+    aws_region=None,
+):
```

Comment:
I honestly it would make more sense and will be more user friendly (and will be easier to maintain for us) to have a simple `with_s3_creds` function, instead of this modifier thing